src/setupWorker/glossary.ts

```diff
@@ -20,3 +22,7 @@ export interface StartOptions {
   }
   quiet?: boolean
 }
+
+export type ResponseWithHeaders = Omit<MockedResponse, 'headers'> & {
```

Comment:
Check where this is used. There's a chance that introduction of a custom `Headers` polyfill negates the need to use this override.